lib/ansible/modules/packaging/language/pip.py

```diff
@@ -235,6 +241,72 @@
                              'pip': 'import pkg_resources; print(pkg_resources.get_distribution("pip").version)'}
 
 
+class Distribution(object):
```

Comment:
You don't need to inherit from `object`, because `__metaclass__ = type` above already turns on new-style classes for everything by default.